projects/plugins/jetpack/extensions/shared/external-media/media-category/index.ts

```diff
@@ -0,0 +1,203 @@
+import apiFetch from '@wordpress/api-fetch';
+import { addQueryArgs } from '@wordpress/url';
+
+// Pexels constants
+const PEXELS_ID = 'pexels';
+const PEXELS_NAME = 'Pexels Free Photos';
+const PEXELS_SEARCH_PLACEHOLDER = `Search ${ PEXELS_NAME }`;
+const DEFAULT_PEXELS_SEARCH: MediaSearch = {
```

Comment:
Nitpick: Can `MediaSerch` be defined above this so developers don't need to interrupt the normal top-to-bottom reading process? Types are usually defined at the very beginning of the file for this reason.

Reply:
You can just navigate to the definition with your IDE's hotkey for doing so.

Reply:
I would accept moving the types to their own file though.

Reply:
> You can just navigate to the definition with your IDE's hotkey for doing so.

Code is not always read with an IDE, but happy to disagree and commit.